scripts/EN/strtab_script

```diff
@@ -1,849 +1,849 @@
+511: ¥"Harry meets Cybil Bennett¥"
+512: ¥"Cybil Bennett¥"
+513: ¥"Cybil holds out the handgun¥"
+514: ¥"Harry receives the gun from Cybil¥"
+515: ¥"Cybil leaves the cafe¥"
+516: ¥"Noise is coming from somewhere…¥"
+517: ¥"Harry picks up the radio¥"
+518: ¥"In this sudden assault, Harry…¥"
+519: ¥"Harry readies the handgun in his inexperienced hands¥"
+520: ¥"Focusing the aim on one thing¥"
+521: ¥"Harry focuses his aim on its body¥"
+522: ¥"This is not a dream…¥"
+523: ¥"Harry heads for the alley¥"
+524: ¥"Cybil shows Harry how to use the gun¥"
+525: ¥"Harry focuses his aim on its wing¥"
+526: ¥"Harry instantly hides¥"
+527: ¥"Harry cannot fight in the tight space indoors¥"
+528: ¥"The radio beyond the window¥"
+529: ¥"Now…¥"
+530: ¥"Back alley¥"
+531: ¥"Along Levin Street¥"
+532: ¥"The advent of further darkness¥"
+533: ¥"School hallway¥"
+534: ¥"Music room¥"
+535: ¥"The grand piano¥"
+536: ¥"The dark courtyard¥"
+537: ¥"Encounter with the oddly shaped monsters¥"
+538: ¥"Harry pushes¥"
+539: ¥"Harry stands paralyzed in amazement¥"
+540: ¥"To the boiler room¥"
+541: ¥"The stopped boiler¥"
+542: ¥"Operating the switchboard¥"
+543: ¥"The clock tower¥"
+544: ¥"Harry fires¥"
+545: ¥"Up the stairs¥"
+546: ¥"The mysterious design¥"
+547: ¥"The transformed school building¥"
+548: ¥"Picking up the receiver¥"
+549: ¥"Calling the police¥"
+550: ¥"Harry after he hung up the telephone receiver¥"
+551: ¥"The open book¥"
+552: ¥"The open book¥"
+553: ¥"Picking up the children's book¥"
+554: ¥"One's own course¥"
+555: ¥"Uncomfortably warm air¥"
+556: ¥"How should I fight…¥"
+557: ¥"Firing the gun at its large body¥"
+558: ¥"Harry suddenly remembers¥"
+559: ¥"Certainly at that time…¥"
+560: ¥"The contents of the children's book¥"
+561: ¥"Aiming at the weak point¥"
+562: ¥"Firing into its mouth¥"
+563: ¥"Harry awakens in a dark room¥"
+564: ¥"A strange feeling of relief¥"
+565: ¥"Calling home, where no one should be¥"
+566: ¥"Picking up the pharmaceutical book¥"
+567: ¥"Picking up the dictionary of religious science¥"
+568: ¥"The contents of the pharmaceutical book¥"
+569: ¥"The contents of the dictionary of religious science¥"
+570: ¥"Stopping its movement first¥"
+571: ¥"Certainly in the Teacher's Room…¥"
+572: ¥"Running is the best bet¥"
+573: ¥"Harry heads for the church¥"
+574: ¥"The sly looking woman¥"
+575: ¥"After receiving the strange information, Harry…¥"
+576: ¥"Harry believes her story¥"
+577: ¥"Left all alone¥"
+578: ¥"The Flauros and a key¥"
+579: ¥"The hospital on the map¥"
+580: ¥"Harry cannot just carelessly believe her¥"
+581: ¥"The location of Alchemilla Hospital¥"
+582: ¥"The hospital wrapped in silence¥"
+583: ¥"Encounter with Kaufmann¥"
+584: ¥"Harry hides behind the door¥"
+585: ¥"Asking about what happened to the town¥"
+586: ¥"The mysterious message¥"
+587: ¥"If there is anything else…¥"
+588: ¥"Harry tries pressing the buttons¥"
+589: ¥"The key on top of the table¥"
+590: ¥"Something is there…¥"
+591: ¥"Pick up the plastic bottle¥"
```

Comment:
Changed pet to plastic to match the PS1 version. (PET refers to the type of plastic and is how they refer to these bottles in Japan.)